docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -0,0 +1,149 @@
+(Optional, boolean)
+If `false`, the request returns an error for any snapshots that are unavailable. Defaults to `false`.
++
+If `true`, the request ignores snapshots that are unavailable, such as those that are corrupted or temporarily cannot be returned.
+
+`verbose`::
+(Optional, boolean)
+If `true`, returns all available information about a snapshot. Defaults to `true`.
++
+If `false`, omits additional information about the snapshot, such as version information, start and end times, and the number of snapshotted shards.
+
+[[get-snapshot-api-example]]
+==== {api-examples-title}
+
+The following request returns information for `snapshot_2` in the `my_repository` repository.
+
+[source,console]
+----
+GET /_snapshot/my_repository/snapshot_2
+----
+
+The API returns the following response:
+
+[source,console-result]
+----
+{
+  "snapshot": {
+    "snapshot": "snapshot_2",
+    "uuid": "vdRctLCxSketdKb54xw67g",
+    "version_id": <version_id>,
+    "version": <version>,
+    "indices": [],
+    "data_streams": [],
+    "include_global_state": false,
+    "metadata": {
+      "taken_by": "user123",
+      "taken_because": "backup before upgrading"
+    },
+    "state": "SUCCESS",
+    "start_time": "2020-06-25T14:00:28.850Z",
+    "start_time_in_millis": 1593093628850,
+    "end_time": "2020-06-25T14:00:28.850Z",
+    "end_time_in_millis": 1593094752018,
+    "duration_in_millis": 0,
+    "failures": [],
+    "shards": {
+      "total": 0,
+      "failed": 0,
+      "successful": 0
+    }
+  }
+}
+----
+// TESTRESPONSE[s/"uuid": "vdRctLCxSketdKb54xw67g"/"uuid": $body.snapshot.uuid/]
+// TESTRESPONSE[s/"version_id": <version_id>/"version_id": $body.snapshot.version_id/]
+// TESTRESPONSE[s/"version": <version>/"version": $body.snapshot.version/]
+// TESTRESPONSE[s/"start_time": "2020-07-06T21:55:18.129Z"/"start_time": $body.snapshot.start_time/]
+// TESTRESPONSE[s/"start_time_in_millis": 1593093628850/"start_time_in_millis": $body.snapshot.start_time_in_millis/]
+// TESTRESPONSE[s/"end_time": "2020-07-06T21:55:18.129Z"/"end_time": $body.snapshot.end_time/]
+// TESTRESPONSE[s/"end_time_in_millis": 1593094752018/"end_time_in_millis": $body.snapshot.end_time_in_millis/]
+// TESTRESPONSE[s/"duration_in_millis": 0/"duration_in_millis": $body.snapshot.duration_in_millis/]
+
+The snapshot `state` can be one of the following values:
+
+[horizontal]
+`IN_PROGRESS`::
+  The snapshot is currently running.
+
+`SUCCESS`::
+  The snapshot finished and all shards were stored successfully.
+
+`FAILED`::
+  The snapshot finished with an error and failed to store any data.
+
+`PARTIAL`::
+  The global cluster state was stored, but data of at least one shard was not stored successfully.
+  The `failures` section of the response contains more detailed information about shards
+  that were not processed correctly.
+
+`INCOMPATIBLE`::
+  The snapshot was created with an old version of {es} and is incompatible with
+  the current version of the cluster.
```

Comment:
The Elastic API reference template includes a section for response body param docs:
https://github.com/elastic/docs/blame/master/shared/api-ref-ex.asciidoc#L123

I think these values would be more appropriate there. As-is, there isn't a definition for `state` on this page.

We also try to avoid use of the `[horizonal]` attribute.

Reply:
I moved these parameters into the `{api-response-body-title}` section and nested them accordingly. Also, removed `[horizontal]`.